### ISSUE TITLE
JoJo Eyes of Heaven regression after #5372

### ISSUE THREAD

**Issue by Youliga** [RPCS3/rpcs3#5534]
JoJo Eyes of Heaven broke after https://github.com/RPCS3/rpcs3/pull/5372 merged, now it 
REDACTED at a 7-10FPS constant. Before it REDACTED at very stable 30FPS and above

**Comment by AniLeo** [comment]
Removed logs as they're from a pirated copy of the game and closing.

If this ever was an issue with the clean copy, it probably isn't in current builds. If someone can reproduce this with a clean copy, please open a new issue with the log for current build.
